app/controllers/UserController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Phalcon\Paginator\Adapter\Model as PaginatorModel;
+
+class UserController extends BaseController{   
+
+    public function listAction(){
+        // Current page to show
+        $currentPage = 1;
+        $currentPage = $this->request->getQuery('page', 'int');
+
+        $users = User::find(
+            array(
+                'order' =>  'created_at DESC'
+            )
+        );
+
+        $paginator = new PaginatorModel(
+            array(
+                "data"  => $users,
```

Comment:
Use a queryBuilder instance instead of model data. In case you're ever to have more users this would highly degrade performance.

Reference: https://docs.phalconphp.com/en/latest/api/Phalcon_Paginator_Adapter_QueryBuilder.html